contracts/Delegations.sol

```diff
@@ -103,31 +114,31 @@ contract Delegations is IDelegations, IStakeChangeNotifier, ContractRegistryAcce
 	function emitDelegatedStakeChangedBatch(address[] memory delegators, uint256[] memory delegatorsStakes, address[] memory delegates) private {
 		uint delegatorsLength = delegators.length;
 
-		address streakDelegate = (delegatorsLength > 0) ? getDelegation(delegators[0]): address(0);
-		uint streakStartIdx = 0;
+		address seqDelegate = (delegatorsLength > 0) ? getDelegation(delegators[0]): address(0);
+		uint seqStartIdx = 0;
 
 		for (uint i = 1; i < delegatorsLength; i++) { // group delegators by delegates. assume sorted by delegate
 			address currentDelegate = delegates[i];
 
-			if (currentDelegate == streakDelegate) {
-				continue; // delegate streak continues
+			if (currentDelegate == seqDelegate) {
+				continue; // delegate seq continues
 			}
 
-			// end of common delegate streak - emit event
-			emitDelegatedStakeChangedSlice(streakDelegate, delegators, delegatorsStakes, streakStartIdx, i - streakStartIdx);
+			// end of common delegate seq - emit event
+			emitDelegatedStakeChangedSlice(seqDelegate, delegators, delegatorsStakes, seqStartIdx, i - seqStartIdx);
 
-			// reset vars for next streak
-			streakDelegate = currentDelegate;
-			streakStartIdx = i;
+			// reset vars for next seq
+			seqDelegate = currentDelegate;
+			seqStartIdx = i;
 		}
 
-		// final streak
+		// final seq
 		if (delegatorsLength > 0) {
-			emitDelegatedStakeChangedSlice(streakDelegate, delegators, delegatorsStakes, streakStartIdx, delegatorsLength - streakStartIdx);
+			emitDelegatedStakeChangedSlice(seqDelegate, delegators, delegatorsStakes, seqStartIdx, delegatorsLength - seqStartIdx);
```

Comment:
this entire function is redundant now that we handle sequences for notifications. move emit to `_stakeChangeBatch`